Reed/Reader/viewmodels/ReaderViewModel.swift

```diff
@@ -62,15 +59,17 @@ class ReaderViewModel: ObservableObject {
     
     func fetchNextSection(sectionNcode: String) {
         fetchSection(sectionNcode: sectionNcode) {
-            self.lastPage = 0
-            self.curPage = 0
+            if self.section?.prevNcode ==  nil {
+                self.curPage = 0
+            } else {
+                self.curPage = 1
+            }
```

Comment:
Nit: Use ternary operator for simple assignments.
`self.curPage = self.section?.prevNcode == nil ? 0 : 1`

Reply:
fixed